bin/metrics-mongodb.rb

```diff
@@ -114,6 +119,7 @@ def run
     collector.connect_mongo_db('admin')
     exit(1) if config[:require_master] && !collector.master?
     metrics = collector.server_metrics
+    metrics = metrics.select { |k, _v| !k[/databaseSizes/] } if config[:exclude_db_sizes]
```

Comment:
Is there any reason to use select and not matching a regex rather than using `.reject` with a positive assertion?

Reply:
No special reason, I did it that way as I'm still learning Ruby. 😃 
If You think it will have better performances I can try to rewrite it?

Reply:
I think its less of a performance thing and more of a readability thing, If you can do that great if not its not a big deal.

Reply:
Please check it out, I hope this is what you mean?

Reply:
Yup just reads a little clearer in its intent imho.

Reply:
Nice, thanks!